third_party/txt/src/txt/paragraph_txt.cc

```diff
@@ -671,7 +682,7 @@ void ParagraphTxt::Layout(double width) {
   glyph_lines_.clear();
   code_unit_runs_.clear();
   inline_placeholder_code_unit_runs_.clear();
-  max_right_ = FLT_MIN;
+  max_right_ = -FLT_MAX;
```

Comment:
I'm not familiar with what this does, is this change related/required by the current change? Will it have any side effects, or is `FLT_MIN` always the `-` of `FLT_MAX`?

Reply:
According to the documentation `FLT_MIN` is the smallest positive `double` (that's not `0`). I had to change this for some strut related stuff so descent can go negative. Some of other occurrences may not be directly related to this change but they all seem to make more sense with `-FLT_MAX`.

Reply:
Yes - I think all of these initializers should have been `-FLT_MAX`.  They are all used in `std::max(current_value, new_value)` expressions, and the goal is to ensure that any `new_value` is greater than the initialized `current_value`.

Another option would be using `std::numeric_limits<double>::lowest()`

Reply:
Switched to std::numeric_limits<T> as they all seem to be `double`s.